accord-core/src/main/java/accord/utils/Timestamped.java

```diff
@@ -68,6 +71,6 @@ public static <T> Timestamped<T> mergeEqual(Timestamped<T> a, Timestamped<T> b,
     @Override
     public String toString()
     {
-        return data.toString();
+        return toString.apply(data);
```

Comment:
`data` is almost always `int[]` which is useless... so logs on error didn't make sense without this change